include/freerdp/client.h

```diff
@@ -41,6 +41,10 @@
 #include <freerdp/client/encomsp.h>
 #endif
 
+#if defined(WITH_SSO_MIB)
+#include <sso-mib/sso-mib.h>
```

Comment:
just remove this include and only include it in `client.c`
wrap the thing in a `typedef struct foo MIBClientAppWrapper ;` (or whatever you deem a good name) and define the struct itself only in `client.c`

this way you do not expose any types from the dependent lib and you only need the include paths/link options for the single file.